mixins/_icons.scss

```diff
@@ -69,16 +73,19 @@
         content: "\64";
     }
     @if $symbol == 'down' {
-        content: "\76";
+        content: "\56";
     }
     @if $symbol == 'download' {
         content: "\2913";
     }
+    @if $symbol == 'expand' {
+        content: "\21f2";
+    }
```

Comment:
Didn't have a symbol for this before, even though it was in the font files. I've added it just in case.